deerhunt/src/pages/AmIAuthPage.js

```diff
@@ -2,27 +2,31 @@ import React from "react";
 import axios from "../config/config";
 
 class Home extends React.Component {
-  constructor(props) {
-    super(props);
 
-    this.all_events = [];
-  }
+    constructor(props) {
+        super(props)
 
-  get_events() {
-    axios
-      .get("http://localhost:5000/api/amiauth")
-      .then((resp) => {
-        console.log(resp.data);
-      })
-      .catch((err) => {
-        console.log(err);
-      });
-  }
+        this.all_events = []
+    }
 
-  render() {
-    console.log("rendering");
-    return <div>{this.get_events()}</div>;
-  }
+
+    get_events(){
+        axios.get("api/amiauth")
+        .then(resp => {
+            console.log(resp.data)
+        }).catch(err => {
+            console.log(err)
+        })
+    }
+
+    render() {
+        console.log("rendering")
+        return (
+            <div>
+                {this.get_events()}
```

Comment:
I recommend using componentWillMount instead of calling function inside the render